server.js

```diff
@@ -74,12 +81,19 @@
     }
 
     var knownTilesetFormats = [/\.b3dm/, /\.pnts/, /\.i3dm/, /\.cmpt/, /\.glb/, /tileset.*\.json$/];
-    app.get(knownTilesetFormats, checkGzipAndNext);
-
-    app.use(function(req, res, next) {
-        res.header("Access-Control-Allow-Origin", "*");
-        res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
-        next();
+    app.use(knownTilesetFormats, checkGzipAndNext);
+
+    // Custom code for serving TilesetWithExpiration. When points.pnts is requested it cycles between the tiles in the cache folder.
+    var expirationPntsPath = '/tilesets/TilesetWithExpiration/points.pnts';
+    var expirationCacheDirectory = '/tilesets/TilesetWithExpiration/cache/';
+    var expirationCacheLength = 5;
+    var expireCount = 0;
+
+    app.use(expirationPntsPath, function(req, res) {
+        var pntsPath = expirationCacheDirectory +  'points_' + expireCount + '.pnts';
+        expireCount = (expireCount + 1) % expirationCacheLength;
+        res.sendFile(pntsPath, {root: __dirname});
+        // Don't call next() because we don't need to run the express.static middleware
```

Comment:
Totally not what I expect it, but I believe it.  Not sure if express and restify have different semantices and I'm more used to rest.  We can just keep this as-is for now then.